traefik/faency
# Update input style

## Issue

https://github.com/traefik/faency/issues/148

## How to test

- Run storybook
- Check stories:
  - Input
  - TextField
  - IconButton => small impact from [commit](2c32c26bf0514159baf5bd4ef4b478c7a67f80e4)

## Design opinion required on
- Ghost variant rework for Input
- Adornments for Input
  - specific layout when there are icon buttons

Rest follows design preco from the issue

## Code review
Don't hesitate to be picky, I'd like to see learn your design patterns

## Blocked by

Blocked by #153 (must add elevation on focused input) and #162
